management-app/src/main/java/com/nearsoft/fgaribay/mgmt/BrokerConnector.java

```diff
@@ -8,26 +8,28 @@
 import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.stereotype.Component;
 
-import java.util.Collections;
-import java.util.List;
+import java.util.ArrayList;
 
 @Component
 public class BrokerConnector {
 
     private final RabbitTemplate rabbitTemplate;
 
-    private ProductProperties.Queues queueProperties;
+    private ProductProperties.Exchanges exchangeProperties;
+    private ProductProperties.RoutingKeys routingKeyProperties;
 
     @Autowired
```

Comment:
`@Autowired` is not needed here since you are using constructor dependency injection. Check it you are using this annotation in other beans and remove it.